<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StackedWindows</class>
 <widget class="QWidget" name="StackedWindows">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1103</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PersonalFinanceManager</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1101</width>
     <height>631</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="EntrancePage">
    <widget class="QLabel" name="labelEPPassword">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonEPOk">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>170</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditEPPassword">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>120</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelEPUserName">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>User Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditEPUserName">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>70</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelEPWarning">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>210</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonEPRegister">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>170</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="RegisterPage">
    <widget class="QLabel" name="labelRPUserName">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>User Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditRPPasswordAgain">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>140</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditRPPassword">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>100</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelRPPasswordAgain">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>140</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>PasswordAgain</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelRPWarning">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>240</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRPRegister">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>190</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditRPUserName">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>50</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelRPPassword">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRPCancel">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>190</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="MainPage">
    <widget class="QChartView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>0</y>
       <width>731</width>
       <height>411</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelMPGreeting">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>420</y>
       <width>431</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Welcome user_name &lt;Will update itself&gt; </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelMPMonthlyRevanue">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>470</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Update Monthly Revanue</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonMPMonthlyRevanueOk">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>470</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMPSpendingCategory">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>530</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Spending Category</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMPMonthlyRevanue">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>470</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMPSpendingCategoryName">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>540</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonMPSpendingCategoryOk">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>540</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMPMonthlyRevanueWarning">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>470</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labelMPSpendingCategoryWarning">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>540</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMPSpendingCategoryValue">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>540</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelMPSpendingCategoryName">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>510</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Category Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMPSpendingCategoryValue">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>510</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Category Value</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMPRemoveSpendingCategory">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>580</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove Spending Category</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMPRemoveSpendingCategoryName">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>580</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonMPRemoveSpendingCategoryOk">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>580</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMPRemoveSpendingCategoryWarning">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>580</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">QtCharts.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditRPUserName</tabstop>
  <tabstop>lineEditRPPassword</tabstop>
  <tabstop>lineEditRPPasswordAgain</tabstop>
  <tabstop>pushButtonRPRegister</tabstop>
  <tabstop>lineEditEPUserName</tabstop>
  <tabstop>lineEditEPPassword</tabstop>
  <tabstop>pushButtonEPOk</tabstop>
  <tabstop>pushButtonEPRegister</tabstop>
  <tabstop>pushButtonRPCancel</tabstop>
  <tabstop>graphicsView</tabstop>
  <tabstop>lineEditMPMonthlyRevanue</tabstop>
  <tabstop>pushButtonMPMonthlyRevanueOk</tabstop>
  <tabstop>lineEditMPSpendingCategoryName</tabstop>
  <tabstop>lineEditMPSpendingCategoryValue</tabstop>
  <tabstop>pushButtonMPSpendingCategoryOk</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
